packages/stark-ui/src/modules/minimap/components/minimap.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="stark-minimap">
+	<div class="stark-minimap-dots">
+		<ul>
+			<li *ngFor="let item of items; trackBy: $index"
+				[ngClass]="{'selected':isItemVisible(item)}">
+				<i></i>
+			</li>
+		</ul>
+	</div>
+	<div class="stark-minimap-dropdown-toggle"
+		 [ngClass]="{'visible':isDisplayedMenu}">
+		<button mat-button class="md-icon-button stark-minimap-dropdown-toggle-button"
+				(click)="toggleDropdownMenu()" 
+				aria-label="Toggle columns"
+				matTooltip="'STARK.TABLE.TOGGLE_COLUMNS'"
+				[matTooltipPosition]="below"
```

Comment:
Currently, Travis is failing because of this line. I think it should be like this: `matTooltipPosition="below"` (without square brackets)